tests/serialized_test_data_generation/make_all_datasets.sh

```diff
@@ -46,28 +46,49 @@ if [ -z "${EXPERIMENTS}" ] ; then
     exit 1
 fi
 
+
 # loop over experiments
 for exp_file in ${EXPERIMENTS} ; do
   exp_name=`basename ${exp_file} .yml`
   echo ""
   echo "++++++++++++++++++++++++++++++++++++++++++++++++++++++"
-  echo "> Generating data for ${exp_name} ..."
+  echo "> Generating driver data for ${exp_name} ..."
   npx=`cat ${exp_file} | grep npx | sed s/npx://g | sed 's/^ *//g'`
-  if [ ${npx} -gt 49 ] ; then
-      export SER_ENV="ONLY_DRIVER"
-  else
-      export SER_ENV="ALL"
+  seconds=`cat ${exp_file} | grep seconds | sed s/seconds://g | sed 's/^ *//g'`
```

Comment:
Script is getting pretty long...probably should refactor to python soon esp given the number of conditional statements and templating logic.

Reply:
It is indeed! I am hesitant to make this python though, it's mildly confusing to have some actions in bash and some in python requiring a virtualenv in the same repo... We will likely continue to revisit this script, so there should be opportunity to pythonize it.

Reply:
No need to change now, but at some point it may be worth paying down the technical debt here otherwise I fear we'll just keep adding to it.